source/reference/json/search-nodes-config-file.txt

```diff
@@ -0,0 +1,94 @@
+.. _atlas-cli-search-nodes-config-file:
+
+===============================
+Search Nodes Configuration File
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To create or update :atlas:`search nodes 
+</cluster-config/multi-cloud-distribution/#std-label-configure-search-nodes>` 
+for a {+cluster+} by using the {+atlas-cli+}, you can use a ``.json`` 
+configuration file to specify the required search nodes settings.
+
+.. _search-nodes-settings:
+
+Search Nodes Settings
+---------------------
+
+You can specify the following settings in the search nodes
+configuration file. For a full list of settings and 
+descriptions, see the :oas-atlas-op:`API specification 
+</createAtlasSearchDeployment>`.
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``specs``
+     - Array of objects
+     - Settings to configure search nodes for your {+cluster+}.
+
+   * - ``specs.instanceSize``
+     - string
+     - Hardware specification for the search node instance sizes.
+       This setting uses the following format:
+
+       - ``<instance-size>_HIGHCPU_NVME``
+       - ``<instance-size>_LOWCPU_NVME``
+
+       For example, ``S20_HIGHCPU_NVME``.
+       To learn more, see :atlas:`Search Nodes Costs
+       </billing/search-node>`.
+
+   * - ``specs.nodeCount``
+     - integer
+     - Number of search nodes in the {+cluster+}.
+
+.. _example-search-nodes-config-file:
+
+Example Search Nodes Configuration File
+---------------------------------------
+
+To create or update search nodes on your {+cluster+}, 
+define the search nodes settings in your |json| file 
+as shown in the following example file:
+
+.. literalinclude:: /includes/create-search-node-config-file.json
+
+Example Search Nodes Configuration Commands
+-------------------------------------------
+
+After you create the file, run the {+atlas-cli+} command 
+to create or update search nodes and specify 
+the ``clusterName`` and ``file`` options. 
+
+Create Search Nodes
+~~~~~~~~~~~~~~~~~~~
+
+The following example creates search nodes for the {+cluster+} named
+``myCluster`` using a JSON configuration file named 
+``search-nodes-config.json``:
+
+.. code-block::
+
+    atlas clusters search nodes create --clusterName myCluster --file /search-nodes-config.json
```

Comment:
if your file is in the same directory it should be either `./search-nodes-config.json` or `search-nodes-config.json`